src/components/checkBoxSelection/checkBoxSelection.scss

```diff
@@ -36,17 +28,62 @@
   margin-top: 20px;
   display: none;
 }
-
 .card-settings {
-  height: 600px;
+  height: 350px;
+  margin-left: 159px;
   overflow-y: scroll;
-  background-color: #b5cfe1;
-  width: 350px;
-  padding-left: 20px;
-  box-shadow: 10px 10px 55px 0px black;
+  background-color:#b5cfe1 ;
+  width: 600px;
+  box-shadow: 10px 10px 55px 0px #000000;
 }
-
 .slider {
   margin-top: 10px;
   margin-bottom: 10px;
 }
+.opt {
+  margin-left: 350px;
+  padding-bottom: 800px;
+}
+.home-display-class {
+  display: flex;
+}
+.choose {
+  width: 50%;
+}
+.display {
+  width: 50%;
+}
+
+.optimize-button{
+  margin-top: 12%;
+  margin-top: 19%;
+  text-align: center;
+  display: inline-block;
+  justify-items: center;
+  width:100px;
+  height: 70px;
+  border-radius: 35px;
```

Comment:
extremely high choice of roundness, most use 25px or less

Reply:
I have reduced my px into 25px.

Reply:
trying to use this example to teach you something ive learnt